gradle/gradle-mvn-push.gradle

```diff
@@ -31,112 +14,67 @@ def getSnapshotRepositoryUrl() {
   return findProperty("SNAPSHOT_REPOSITORY_URL") ?: "https://oss.jfrog.org/artifactory/oss-snapshot-local"
 }
 
-def getRepositoryUsername() {
-  return findProperty("BINTRAY_USER") ?: "no.bintray.user"
+task sourcesJar(type: Jar, dependsOn: classes) {
+  archiveClassifier = "sources"
+  from sourceSets.main.allSource, "build/generated/source/kapt/main", "build/generated/source/kaptKotlin/main"
 }
 
-def getRepositoryPassword() {
-  return findProperty("BINTRAY_API_KEY") ?: "no.bintray.api.key"
+task javadocJar(type: Jar, dependsOn: javadoc) {
```

Comment:
How does this play with kdocs?

Reply:
We have just the config but no doc added yet

Reply:
Ok, so I assume you'll circle back to double check this whenever you add kdocs?